bin/create_dynamodb_config

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env ruby
+require 'exercism_config'
+require 'exercism_config'
+
+def create_table(client)
+  client.create_table(
+    table_name: :config,
+    attribute_definitions: [
+      {
+        attribute_name: "id", 
+        attribute_type: "S", 
+      },
+    ],
+    key_schema: [
+      {
+        attribute_name: "id", 
+        key_type: "HASH", 
+      }
+    ],
+    provisioned_throughput: {
+      read_capacity_units: 1, 
+      write_capacity_units: 1, 
+    }
+  )
+end
+
+def delete_table(client)
+  client.delete_table(
+    table_name: :config,
+  )
+end
+
+def set_config_value(client, id, value)
+  client.put_item(
+    table_name: :config,
+    item: {
+      id: id,
+      value: value
+    }
+  )
+end
+
+client = ExercismConfig::SetupDynamoDBClient.()
+
+begin
+  create_table(client)
+rescue Aws::DynamoDB::Errors::ResourceInUseException => e
+  delete_table(client)
+  create_table(client)
+end
+
+settings = {
+  aws_iterations_bucket: "exercism-staging-iterations",
+  dynamodb_tooling_jobs_table: "tooling_jobs"
+}
+if ENV["EXERCISM_DOCKER"]
+  settings.merge!(
+    tooling_orchestrator_url: "http://tooling-orchestrator:3021",
+    anycable_endpoint: 'redis',
+    anycable_rpc_host: '0.0.0.0',
+    dynamodb_endpoint: "http://localhost:8000",
+    rds_master_endpoint: "mysql",
+    rds_port: 3306
+  )
+else
+  settings.merge!(
+    tooling_orchestrator_url: "http://127.0.0.1:3021",
+    anycable_endpoint: '127.0.0.1',
+    anycable_rpc_host: '127.0.0.1',
+    dynamodb_endpoint: "http://localhost:3039",
+    rds_master_endpoint: "localhost",
+    mysql_socket: "/tmp/mysql.sock"
+  )
+end
```

Comment:
@ErikSchierboom This is the key bit. So any of these settings can be used in any Exercism application by doing:
```
Exercism.config. tooling_orchestrator_url
```

This script needs to be called when the Docker starts (which it is) in dev, and never in production (I'll add a guard for that next).

But crucially, if you need to change any settings for Docker/local to work, this is the place to do it.

Reply:
(cc @kntsoriano to this too - I'll talk you through it all properly when it's done and you can put your head in Exercism-space - but nice for you to see now :))